packages/server/src/server.ts

```diff
@@ -39,11 +53,58 @@ const buildApp = (): Promise<Express> => {
     app.use(loggingMiddleware);
     app.use(bodyParser.urlencoded({ extended: true }));
 
+    // Initialise dependencies
+    const [
+        channelSwitches,
+        superModeArticles,
+        articleEpicTests,
+        liveblogEpicTests,
+        holdbackEpicTests,
+        ampEpicTests,
+        choiceCardAmounts,
+        tickerData,
+        productPrices,
+        bannerTests,
+        bannerDeployTimes,
+        headerTests,
+    ] = await Promise.all([
```

Comment:
This is definitely a contender for biggest `Promise.all` array I've ever seen! 😄